apps/sms/js/information.js

```diff
@@ -149,7 +149,9 @@ function showSimInfo(element, iccId) {
   var info = [];
   var simId = Settings.getServiceIdByIccId(iccId);
```

Comment:
After some experiment, both simId and operator will unable to be fetched if the iccId could not match any sim in the slot and the detailString will always empty. Maybe we can fetch the icc first and early return if not exist?